### DISCUSSION TITLE
Branding: is OH!SHOWN a good "logo" wording?

### DISCUSSION THREAD

**Discussion by tai271828** [OH-SHOWN/ohshown-frontend#56] [General]
The other candidates like:

- `OH!SHOWN` the proposed one
- `OH SHOWN!` the exclamation mark is at the end
- `OH!SHOWN!` two exclamation marks
- `Oh!Shown` mixed with lower cases

**Comment by tai271828** [comment]
I noticed that one of our image is `OH SHOWN!` https://github.com/tai271828/disfactory-frontend/blob/main-tbbca/public/images/oh-shown-logo.png XDDD Maybe we need to unify them someday.

**Reply by diaoga** [reply]
Maybe @joshuay1  could help fix it?

> I noticed that one of our image is `OH SHOWN!` https://github.com/tai271828/disfactory-frontend/blob/main-tbbca/public/images/oh-shown-logo.png XDDD Maybe we need to unify them someday.

**Reply by joshuay1** [reply]
Sure I'll work on that. Thanks for pointing this out!
